src/main/java/com/spotify/futures/CompletableFuturesExtra.java

```diff
@@ -55,4 +60,84 @@ private CompletableFuturesExtra() {
     }
     return new CompletableToListenableFutureWrapper<V>(future);
   }
+
+  public static <T> CompletableFuture<T> immediatelyFailed(Throwable throwable) {
+    final CompletableFuture<T> future = new CompletableFuture<T>();
+    future.completeExceptionally(throwable);
+    return future;
+  }
+
+  public static <T> CompletableFuture<T> exceptionallyCompose(
```

Comment:
What's the rationale for tying the API to always returning a CompletableFuture, when all it needs as inputs is CompletionStages? That prevents changing the implementation to in the future return some other kind of CompletionStage.